0036-spsp-pull-payments/0036-spsp-pull-payments.md

```diff
@@ -0,0 +1,135 @@
+---
+title: SPSP Pull Payments
+draft: 1
+---
+# SPSP Pull Payments
+
+## Preface
+
+This document describes how to conduct pull payments via [STREAM](../0029-stream/0029-stream.md) once the payment details have been exchanged via the [Simple Payment Setup Protocol](../0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md) (SPSP).
```

Comment:
what's a pull payment? some readers might not be familiar with the term or what it means in an ILP context

Reply:
I'll define it.